infinispan/runtime/src/main/java/io/gingersnapproject/infinispan/runtime/EmbeddedCacheManagerProducer.java

```diff
@@ -3,29 +3,39 @@
 import javax.annotation.PreDestroy;
 import javax.enterprise.context.ApplicationScoped;
 import javax.enterprise.inject.Produces;
+import javax.inject.Inject;
 import javax.inject.Singleton;
 
+import org.infinispan.configuration.global.GlobalConfiguration;
 import org.infinispan.configuration.global.GlobalConfigurationBuilder;
 import org.infinispan.globalstate.ConfigurationStorage;
 import org.infinispan.manager.DefaultCacheManager;
 import org.infinispan.manager.EmbeddedCacheManager;
+import org.infinispan.metrics.config.MicrometerMeterRegisterConfigurationBuilder;
+
+import io.micrometer.core.instrument.MeterRegistry;
 
 @ApplicationScoped
 public class EmbeddedCacheManagerProducer {
 
-   volatile EmbeddedCacheManager cacheManager;
+   private volatile EmbeddedCacheManager cacheManager;
+   @Inject
+   MeterRegistry meterRegistry;
+
+   private GlobalConfiguration globalConfiguration() {
+      GlobalConfigurationBuilder builder = new GlobalConfigurationBuilder();
+      builder.globalState().enable().configurationStorage(ConfigurationStorage.VOLATILE);
+      builder.cacheContainer().statistics(true);
+      builder.metrics().gauges(true);
+      builder.addModule(MicrometerMeterRegisterConfigurationBuilder.class).meterRegistry(meterRegistry);
+      return builder.build();
```

Comment:
Unfortunately we can't use this approach now that https://github.com/gingersnap-project/cache-manager/pull/5 has been merged.

Reply:
looks like we are not using Infinispan anymore :)